.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
```

Comment:
You have done a great job adding the `.eslintrc` file but you haven't done the installation. don't forget that it is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) that this project has the eslint checked among the linters. 😉
![linters_reqr](https://user-images.githubusercontent.com/64643995/215539209-7bb23700-0e04-40d6-8d76-addec04e6684.png)

Your linter check is currently missing the eslint check
![Screenshot from 2023-01-30 17-42-49](https://user-images.githubusercontent.com/64643995/215539892-1c9e48e0-10a4-49cc-a3e4-ce65b9f612fc.png)

Kindly make the necessary adjustments😉